src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/HashHelper.cs

```diff
@@ -2,15 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-//
-//
-//
 // Description: Static class to help work around hashing-related bugs.
-//
 
-using System;
-using MS.Internal;                  // BaseHashHelper
 
+/* Unmerged change from project 'PresentationFramework'
+Removed:
```

Comment:
nit: removal instead of comment?

Reply:
Rewrote this class completely as it was only ever accessed in PresentationFramework. Now there is a much simpler single `HashHelper` class.